README.md

```diff
@@ -178,8 +169,53 @@ PGADMIN_VERSION=8.9
 This variable sets the version of pgAdmin 4 to be used. The Docker Compose file references
 this variable to pull the corresponding pgAdmin 4 Docker image.
 
-Using these version parameters in the `.env` file allows for easy updates and consistent configuration across 
-different setups.
+Using these version parameters in the `.env` file allows for easy updates and consistent configuration across different setups.
+
+
+## 1.4 Start the Docker compose environment
+
+The Compose file, or `compose.yaml` file, follows the rules provided by the Compose Specification in how to define multi-container applications.
+
+The default path for a Compose file is `compose.yaml` which is the prefered syntax. There are also other options such as the `compose.yml`, which is placed in the working directory.
+
+With Compose there are also **still** supported options like the `docker-compose.yaml` and docker-compose.yml for backwards compatibility of earlier versions. If both files exist, Compose prefers the canonical `compose.yaml.`
+
+#### Key commands
+
+To start all the services defined in your `compose.yaml`file:
+
+
+    $ docker compose up
+
+  This version however opens all the logs, which can be confusing.
+  >**Info** With a slight chance in the command the logs can be observed easily. If any chances are made, look up the updated tutorials on https://docs.docker.com/compose/compose-application-model/
```

Comment:
Use "change" instead of "chance", remove whitespace at the beginning of the paragraph. But I don't really understand the information in this context.